backend/core/src/main.rs

```diff
@@ -103,6 +105,27 @@ async fn node_route(
     )))
 }
 
```

Comment:
```suggestion
Entry point for submitting a shard
```

Reply:
I think this endpoint is more like `/// Endpoint for shards to submit data`.

Reply:
Updated, thanks!